### PR TITLE
Added "what is openstack page" and logo grid pattern

### LINKED ISSUE #4083: "what is openstack?" page - build
https://docs.google.com/document/d/1VL0EroWuIMiATSYysssDyY0ngv48QhniuypmgUpsUWI/edit

### PR DESCRIPTION
**Pull request by shivjoshi1996**
## Done

- Added a new "What is OpenStack" page
- Updated the navigation YAML file
- Added new grid pattern style for responsive logos

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/openstack/what-is-openstack](http://0.0.0.0:8001/openstack/what-is-openstack)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Check the page matches the [copydoc](https://docs.google.com/document/d/15xrgIq_KUwHaCyFoI7-yS_JmBbsoyXJ4GpRMFAa7_is/edit) (The diagram will be added next iteration).

## Issue / Card

Fixes #4083

### CODE REVIEW

**Comment by caldav** [comment]
@shivjoshi1996 There are some (small, won't need a layout change) things missing from the copy doc, that were discussed, but apparently not added. I'll try to get it updated today, sorry about that.

**Comment by shivjoshi1996** [comment]
@caldav No problem, let me know when you've made the changes

**Comment by caldav** [comment]
@shivjoshi1996 thanks for the update! :+1:

**Comment by shivjoshi1996** [comment]
@caldav All sorted, ready to be reviewed @inuwa

File: templates/openstack/what-is-openstack.html [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+  </div>
+</section>
+
+<section class ="p-strip--light is-bordered">
+  <div class="row">
+    <div class="col-8">
+      <h2>What makes up OpenStack?</h2>
+      <p>At its most basic, OpenStack is comprised of storage, compute, and networking projects, such as the following core examples...</p>
+    </div>
+  <div>
```
Comment (inuwa) [inline templates/openstack/what-is-openstack.html]: Div should be a closing tag

File: templates/openstack/what-is-openstack.html [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+        <p>OpenStack was originally launched in July 2010 by Rackspace and NASA as an open source initiative that combined NASA’s Nebula platform and Rackspace’s Cloud Files platform.</p>
+        <p>Today, hundreds of companies contribute to OpenStack code, with many more playing a part in the broader community.</p>
+      </div>
+    </div>
+</section>
+
+<section class="p-strip u-no-padding--top u-no-padding--bottom">
+  <div class="col-12"> 
+    <ul class="row"> 
+      <h3 class="p-muted-heading">Influential contributors to openStack</h3>
```
Comment (inuwa) [inline templates/openstack/what-is-openstack.html]: The h3 tag fails validation. Apparently it is not allowed a child UL. You might need to move it up.

**Pull request merged**